### DISCUSSION TITLE
Arduino IoT support

### DISCUSSION THREAD

**Discussion by zuloomi** [supabase/supabase#3410] [Questions]
There's a big maker community out there building IoT systems via Arduino which uses C++. I haven't seen any C++ code samples on Supabase.

Has anyone utilised Supabase in their Arduino projects? If so how?

**Comment by awalias** [comment]
that sounds awesome! 

fyi the raw api is just a http endpoint `https://<your-project-ref>.supabase.in/rest/v1/table_name` that you can GET, POST, PATCH, DELETE etc. using the postgrest interface here https://postgrest.org/en/v8.0/api.html#horizontal-filtering-rows - so it should be fairly straight forward to start querying your data (in any language with nice http request support)

**Reply by zuloomi** [reply]
How could I miss that :) Had a quick look at that part of the doc and found some samples to adopt already. Will get cracking on this. txs!

**Reply by steve-chavez** [reply]
@zuloomi I suggest looking at [postgrest-js](https://github.com/supabase/postgrest-js) for some inspiration :+1:
